Sources/Noora/NooraMock.swift

```diff
@@ -0,0 +1,53 @@
+#if DEBUG
+    import Rainbow
+
+    public class MockNoora: Noora, CustomStringConvertible {
```

Comment:
Very nitpick, but I'd go for:

```suggestion
    public class NooraMock: Noora, CustomStringConvertible {
```